scss/_footer.scss

```diff
@@ -0,0 +1,72 @@
+body .container {
```

Comment:
@laurxnemeth This is actually overriding another style that it seems we import: https://github.com/NYULibraries/statuspage-embed/blob/master/scss/index.scss#L8

Could you work with @ericgriffis to see if we can resolve that there so we're not overriding our own styles? It may prove you can't and we have to but an avenue to explore first

Reply:
@barnabyalter @laurxnemeth I went ahead and removed that style from statuspage-embed, I didn't think that should putting styles on `body` and I don't know why it was in the first place since removing them didn't seem to change it much if at all.